PCA_shear.py

```diff
@@ -0,0 +1,161 @@
+import sys
+sys.path.insert(0,'../')
+from cov_3X2 import *
+
+# only for python3
+import importlib
+reload=importlib.reload
+
+
+
+class PCA_shear():
+    def __init__(self,NmarQ,PCA_simName=None,kappa_class=None,clS=None):
```

Comment:
```suggestion
    def __init__(self,NmarQ,kappa_class,PCA_simName=None,clS=None):
```
`kappa_class` does not seem to have a default.